### PR TITLE
Feature/6454 register vaccination certificates/6937 info consent screen

### PR DESCRIPTION
**Pull request by KaiTeuber**
## Description
HealthCertificate Consent screen, will get shown if the user will create a certificate

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6937

## Screenshots
Simulator Screen Shot - iPhone 12 Pro - 2021-05-06 at 15 32 47
Simulator Screen Shot - iPhone 12 Pro - 2021-05-06 at 15 32 52
Simulator Screen Shot - iPhone 12 Pro - 2021-05-06 at 15 32 59
Simulator Screen Shot - iPhone 12 Pro - 2021-05-06 at 15 33 03

### CODE REVIEW

**Review by whiskey — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings [hunk 1]
```diff
@@ -195,3 +195,19 @@
...
 
 "ExposureSubmission_Antigen_Profile_Legal_Text07" = "Private Personen oder Unternehmen dürfen von Ihnen nicht verlangen, dass Sie Ihr Schnelltest-Profil vorzeigen.";
+
+/* Health Certificate Information */
+
+"HealthCertificate_Info_Legal_headline" = "Ihr Einverständnis";
+
+"HealthCertificate_Info_Legal_subHeadline" = "Durch Antippen von „Einverstanden“ willigen Sie in folgende Schritte ein:";
+
+"HealthCertificate_Info_Legal_section01" = "Die App legal";
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings:205]: Wat? :D

**Review by marcussc — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings (hunk 1 above)
Comment (KaiTeuber) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings:205]: the legal texts are not final at the moment

Comment (marcussc) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.legal.strings:205]: Ah okay 👍

**Review by NataliaLemmerth — COMMENTED** [review]
UA approved, texts are not final

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificate/Consent/HealthCertificateConsentViewController.swift [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	// MARK: - Protocol FooterViewHandling
+
+	func didTapFooterViewButton(_ type: FooterViewModel.ButtonType) {
+		guard type == .primary else {
+			return
+		}
+		didTapConsentButton()
+	}
+
+	// MARK: - Public
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificate/Consent/HealthCertificateConsentViewController.swift:53]: left on purpose?

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificate/Consent/__tests__/HealthCertificateConsentViewModelTests.swift [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import XCTest
+@testable import ENA
+
+class HealthCertificateConsentViewModelTests: XCTestCase {
+
+	func testGIVEN_ConsentViewModel_WHEN_getDynamicTableViewModel_THEN_CellsAndSectionsCountAreCorrect() {
+		// GIVEN
+		let viewModel = HealthCertificateConsentViewModel(didTapDataPrivacy: {})
+
+		// WEHEN
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificate/Consent/__tests__/HealthCertificateConsentViewModelTests.swift:14]: When

**Review by NataliaLemmerth — APPROVED** [review]

**Pull request merged**